src/components/FormSelectList/OptionboxView.vue

```diff
@@ -1,91 +1,102 @@
 
 const uniqIdsMixin = createUniqIdsMixin();
 
 export default {
-  inheritAttrs: false,
   mixins: [uniqIdsMixin, ValidationMixin],
+  inheritAttrs: false,
   props: [
-    'value',
-    'optionValue',
-    'optionContent',
-    'options',
-    'error',
-    'helper',
-    'name',
-    'controlClass',
-    'emitObjects',
-    'emitArray',
-    'optionAriaLabel'
+    "value",
+    "optionValue",
+    "optionContent",
+    "options",
+    "optionsExtra",
+    "error",
+    "helper",
+    "name",
+    "controlClass",
+    "emitObjects",
+    "emitArray",
+    "optionAriaLabel"
   ],
   data() {
     return {
-      selected:[]
-    }
-  },
-  mounted() {
-    this.selected = this.value;
-  },
-  watch: {
-    value(val) {
-      this.selected = val;
-    },
-    selected() {
-      this.$emit('input', this.selected);
-    }
+      selected: []
+    };
   },
   computed: {
     divClass() {
-      return this.toggle ? 'custom-control custom-radio' : 'form-check';
+      return this.toggle ? "custom-control custom-radio" : "form-check";
     },
     labelClass() {
-      return this.toggle ? 'custom-control-label' : 'form-check-label';
+      return this.toggle ? "custom-control-label" : "form-check-label";
     },
     inputClass() {
       return [
-        {[this.controlClass]: !!this.controlClass},
-        {'is-invalid': (this.validator && this.validator.errorCount) || this.error},
-        this.toggle ? 'custom-control-input' : 'form-check-input'
+        { [this.controlClass]: !!this.controlClass },
+        { "is-invalid": (this.validator && this.validator.errorCount) || this.error },
+        this.toggle ? "custom-control-input" : "form-check-input"
       ];
+    }
+  },
+  watch: {
+    value(val) {
+      this.selected = val;
     },
+    selected() {
+      this.$emit("input", this.selected);
+    }
+  },
+  mounted() {
+    this.selected = this.value;
   },
   methods: {
     getOptionValue(option) {
-      return option[this.optionValue || 'value'];
+      return option[this.optionValue || "value"];
     },
     getOptionContent(option) {
-      return option[this.optionContent || 'content'];
+      return option[this.optionContent || "content"];
     },
     getOptionAriaLabel(option) {
-      const ariaLabel = option[this.optionAriaLabel || "ariaLabel"];
-      return (!ariaLabel || ariaLabel === "") ? this.getOptionContent(option) : ariaLabel;
+      let ariaLabel = "";
+      if (this.optionsExtra?.length) {
+        const optionExtra = this.optionsExtra.find(
+          (extra) => extra.hasOwnProperty(this.optionValue) && extra[this.optionValue] === option[this.optionValue]
+        );
+        if (optionExtra) {
+          ariaLabel = optionExtra[this.optionAriaLabel || "ariaLabel"] ?? "";
+        }
+      } else {
+        ariaLabel = option[this.optionAriaLabel || "ariaLabel"] ?? "";
+      }
+      return !ariaLabel || ariaLabel === "" ? this.getOptionContent(option) : ariaLabel;
```

Comment:
Please change this line to: `return ariaLabel?.trim() ? ariaLabel : this.getOptionContent(option);`  This ensures that an ariaLabel with only spaces is also considered "empty".

Reply:
This line of code allows a string longer than one character with spaces to be considered valid. Is that correct?